modules/userId/index.js

```diff
@@ -197,19 +197,32 @@ function processSubmoduleCallbacks(submodules) {
 }
 
 /**
- * @param {AdUnit[]} adUnits
+ * This function will create a combined object for all subModule Ids
  * @param {SubmoduleContainer[]} submodules
  */
-function addIdDataToAdUnitBids(adUnits, submodules) {
-  if ([adUnits, submodules].some(i => !Array.isArray(i) || !i.length)) {
-    return;
+function getCombinedSubmoduleIds(submodules) {
+  if ([submodules].some(i => !Array.isArray(i) || !i.length)) {
```

Comment:
Since only submodules is being iterated now, change to:
`if (!Array.isArray(submodules) || !submodules.length) {`

Reply:
done.